### PR TITLE
[Frontend] Integration of API for keeping track of collaborations with brandprofile page and making page responsive fixed

### LINKED ISSUE #52: [Frontend] Integration of API for keeping track of collaborations with brandprofile page and making page responsive
Fetching data from api and integrating it on brandprofile page and refactor the code

### PR DESCRIPTION
**Pull request by Hemu21**
## Description

Brand profile page is responsive and refactored, collabration integration was done.

## Issue

[Issue Link](https://github.com/Kajol-Kumari/InfluMart/issues/52) resolve #52

### CODE REVIEW

**Comment by Hemu21** [comment]
@Kajol-Kumari  @RohitGupta1237  can you please check

**Comment by RohitGupta1237** [comment]
@Hemu21  
Screenshot 2024-06-25 at 3 40 29 PM

hey other items are not showing even if no deal is there show 0 or NA in those fields

**Comment by Hemu21** [comment]
@RohitGupta1237  updated with requested changes.

File: backend/routes/collaborationRoutes.js [hunk 1]
```diff
@@ -8,17 +8,15 @@ const {
...
 router.post('/collaborations', collaborationAuthenticationMiddleware,
     collaborationAuthorizationMiddleware, createCollaboration);
 
-router.get('/collaborations/:brandId', collaborationAuthenticationMiddleware,
-    collaborationAuthorizationMiddleware, getBrandCollaborations);
+router.get('/collaborations/:brandId', brandAuthenticationMiddleware, getBrandCollaborations);
 
-router.get('/minimum-requirements/:brandId', collaborationAuthenticationMiddleware,
-    collaborationAuthorizationMiddleware, getBrandMinimumRequirements);
+router.get('/minimum-requirements/:brandId', brandAuthenticationMiddleware, getBrandMinimumRequirements);
```
Comment (Kajol-Kumari) [inline backend/routes/collaborationRoutes.js:18]: collaborationAuthenticationMiddleware is removed intentionally?

**Review by Kajol-Kumari — APPROVED** [review]

**Pull request merged**